app/views/users/update.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Information updated</h1>
```

Comment:
- [ ] **[OPTIONAL]** Instead of rendering a page for success message "information updated", we can use [flash messages](http://stevenleiva.com/flash-messages-in-rails).